src/main/kotlin/no/nav/klage/oppgave/repositories/BehandlingRepository.kt

```diff
@@ -24,17 +23,14 @@ interface BehandlingRepository : JpaRepository<Behandling, UUID>, JpaSpecificati
 
     fun findByAvsluttetAvSaksbehandlerIsNullAndFeilregistreringIsNull(): List<Behandling>
 
-    @EntityGraph(attributePaths = ["saksdokumenter", "hjemler", "registreringshjemler", "medunderskriverHistorikk"])
     fun findByTildelingEnhetAndAvsluttetAvSaksbehandlerIsNullAndFeilregistreringIsNull(enhet: String): List<Behandling>
 
     fun findByIdAndAvsluttetIsNotNull(id: UUID): Behandling?
 
     fun findBySakenGjelderPartIdValueAndAvsluttetAvSaksbehandlerIsNullAndFeilregistreringIsNull(partIdValue: String): List<Behandling>
 
-    @EntityGraph(attributePaths = ["saksdokumenter", "hjemler", "registreringshjemler", "medunderskriverHistorikk"])
-    override fun findAll(
-        specification: Specification<Behandling>,
-        sort: Sort,
-    ): List<Behandling>
+    @EntityGraph("Behandling.full")
+    @Query("select b from Behandling b where b.id = :id")
```

Comment:
Hvorfor trengs eksplisitt Query her? Er det for å slippe å overstyre `findById`?

Reply:
Ja, `findById` er allerede i bruk.